RMtools_3D/do_fitIcube.py

```diff
@@ -70,77 +77,122 @@ def main():
                                  formatter_class=argparse.RawTextHelpFormatter)
     parser.add_argument("fitsI", metavar="StokesI.fits", nargs=1,
                         help="FITS cube containing Stokes I data.")
-    parser.add_argument("freqFile", metavar="freqs_Hz.dat", nargs=1,
-                        help="ASCII file containing the frequency vector.")
```

Comment:
Good point, added back as an option. If not provided, the code will derive the frequency axis using fits header. This is printed out so that the user is aware what is happening. And is also mentioned under -h.